### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,55 +12,39 @@ inference.
...
+* [Instance Segmentation](models/instance_segmentation)
+  - [COCO instance segmentation](models/instance_segmentation/model_templates/coco-instance-segmentation/readme.md)
+* [Object Detection](models/object_detection)
+  - [Face Detection](models/object_detection/model_templates/face-detection)
+  - [Horizontal Text Detection](models/object_detection/model_templates/horizontal-text-detection/)
+  - [Person Detection](models/object_detection/model_templates/person-detection/)
+  - [Person Vehicle Bike Detection](models/object_detection/model_templates/person-vehicle-bike-detection)
+  - [Vehicle Detection](models/object_detection/model_templates/vehicle-detection)
+* [Text Spotting](models/text_spotting)
+  - [Alphanumeric Text Spotting](models/text_spotting/model_templates/alphanumeric-text-spotting/readme.md)
```
Comment (sovrasov) [inline README.md:43]: Where is object classification?

Comment (Ilya-Krylov) [reply README.md:45]: thx, added

File: README.md [hunk 2]
```diff
@@ -12,55 +12,44 @@ inference.
...
-  * [Text Recognition](tensorflow_toolkit/text_recognition)
-  * [Vehicle Attributes](tensorflow_toolkit/vehicle_attributes)
-  * [Bert](tensorflow_toolkit/bert)
-
----
-\* Other names and brands may be claimed as the property of others.
+After installation, you are ready to train your own models, evaluate and use
+them for prediction.
+
+* [Action Recognition](models/action_recognition_2)
```
Comment (evgeny-izutov) [inline README.md:35]: Maybe it's good time to rename it back to 'models/action_recognition' to resolve possible questions like `where is the first action_recognition here`?

Comment (Ilya-Krylov) [reply README.md:35]: definitely, but not right now, I would minimize changes in single PR as much as possible

**Pull request merged**